.gitmodules

```diff
@@ -28,7 +28,7 @@
 	url = https://github.com/PX4/Matrix.git
 [submodule "src/lib/DriverFramework"]
 	path = src/lib/DriverFramework
-	url = https://github.com/PX4/DriverFramework.git
+	url = git@github.com:3drobotics/DriverFramework
```

Comment:
@bugobliterator Can you please make a proper pull request of your DriverFramework changes against [PX4/DriverFramework](https://github.com/PX4/DriverFramework)?

It's quite painful to point to different submodules, so let's try to avoid this.